### DISCUSSION TITLE
Very long time stuck at 100%

### DISCUSSION THREAD

**Discussion by sawoodpearce** [mifi/lossless-cut#1370] [Support]
In recent versions of Lossless Cut, for some reason the tool gets stuck at 'Exporting 100%' for a number of minutes. Eventually it says done. It's not a major issue, but it didn't do it before.

Is this an issue with my local machine? I am using it to trim MPEG2 files. Screenshot attached.

![2022-11-12 08_20_50-LosslessCut](https://user-images.githubusercontent.com/97291912/201465378-88f398ab-b7b2-4728-9041-e7ba40373cdc.png)

I guess technically...it's not really 100% done :)

Thank you.

**Comment by mifi** [comment]
Hi. Is it export+merge? do you see anything in the javascript console right before or while it's stuck at 100%?

**Reply by sawoodpearce** [reply]
Apologies for my delayed reply. This is export only, trimming a single clip. I have taken a screenshot of the console while it's stuck at 100%, attached. It happens with every file. 

Win 7 x64, v3.47.1.

![2023-01-08 18_56_15-LosslessCut - Reece T04 mpg](https://user-images.githubusercontent.com/97291912/211213814-4f314c5f-f007-4115-8e74-64b557674a7c.png)

**Reply by mifi** [reply]
Thanks. It's probably one of either:
- a bug in ffmpeg where it's not properly reporting progress
- duration is wrong, so calculation is wrong. is it correct?
- fps is wrong. is 25 correct?
- a bug in losslesscut

does it happen also with for example mp4/h264 files? I need to find a file that can reproduce the problem

**Reply by sawoodpearce** [reply]
1 & 4. Based on CPU usage the job appears to be still processing while reporting 100%.
2. To the nearest second, the duration of the output files matches the 'Segments total' shown in the application.
3. Input and output should be 25fps for this footage and appear to be.

This workflow has MPEG2 files as the input, so Losslesscut is creating a proxy on load. I could email you a link to a file which behaves like this, if that helps.

**Reply by mifi** [reply]
If you can email a link to an original file to losslesscut@yankee.no that would be helpful

**Reply by mifi** [reply]
Thanks for sending over. I see that it takes a bit of extra time once it reaches 100%. For me it's just a few seconds though, maybe my macbook air m2 is just blazingly fast 😅
I think it's caused by the fact that your video has an extra track of type `dvd_nav_packet`, which is not able to be processed by losslesscut, so it will instead extract it as a separate file. This is when the progress is stuck at 100% while extracting that track. I will improve the situation by showing to the user what's happening at that time. You can go to the Tracks panel and choose the Discard unprocessable tracks option at the bottom to prevent this.

**Comment by sawoodpearce** [comment]
Thanks for the very quick reply, Mikael.

My standalone capture station is an i5 running Windows 7 :D so it's longer. Then it heads off to my server to be encoded. I'm glad you even spotted it on an M2 though.

These are native files from the VHS capture software. Thanks for the suggested workflow step to improve the accuracy meanwhile.

Thanks,
Sawood


--

Sawood Pearce

Deloitte Digital | Deloitte MCS Limited

D: +44 20 7007 3344 | M: +44 7827 367130

***@***.******@***.***> | www.deloitte.co.uk<file:///private/var/containers/Bundle/Application/A4FB7B8C-BAB7-40AE-B5AF-E11A19F29A5E/Outlook-iOS.app/Frameworks/EmailRendererKit.framework/www.deloitte.co.uk>

I am currently on planned leave most Fridays.



<https://aka.ms/o0ukef>
________________________________
From: Mikael Finstad ***@***.***>
Sent: Monday, January 16, 2023 4:33 am
To: mifi/lossless-cut ***@***.***>
Cc: Pearce, Sawood ***@***.***>; Author ***@***.***>
Subject: [EXT] Re: [mifi/lossless-cut] Very long time stuck at 100% (Discussion #1370)


Thanks for sending over. I see that it takes a bit of extra time once it reaches 100%. For me it's just a few seconds though, maybe my macbook air m2 is just blazingly fast 😅
I think it's caused by the fact that your video has an extra track of type dvd_nav_packet, which is not able to be processed by losslesscut, so it will instead extract it as a separate file. This is when the progress is stuck at 100% while extracting that track. I will improve the situation by showing to the user what's happening at that time. You can go to the Tracks panel and choose the Discard unprocessable tracks option at the bottom to prevent this.

—
Reply to this email directly, view it on GitHub<https://secure-web.cisco.com/1CTvv8QN4I4LLSZZ-KMRyjBux6OzlFyrWUV_vpsGxcfpm1qZC0yzydPpu-w1a7ZreOe6J6Saq-BW-ZqP9H7PR0il5f7sQEfw4XiqrceI698n3uwLDijaJfGPKRWpr_E4fTgQHv-cvjlre-irAydazRFN6KT9B5lUPIav7_WFGYJvrivHIrCr6PmcUeZt22zp9lbfAxI2uez3JFMz_qgAHSjrRkxTzi847fK2VeLrbpAJAeaiKM1EU3vzU-h7DAs2mbM_0aZJAagA8LJHVHH8LXlaNJm71getM5yIuzuPH5e-AXv-DVn6IiD01eF1XJVcE/https%3A%2F%2Fgithub.com%2Fmifi%2Flossless-cut%2Fdiscussions%2F1370%23discussioncomment-4693579>, or unsubscribe<https://secure-web.cisco.com/1U1WDknJ6LgMQA1L0WDSbMb3HbgauvNRy6cIn1mlFq84q5NlIDIED2FeH4zQyxkkE-FzSWbgY07WBBR_Zt3T3EqFzM7tElpF2eP7rHIIB8fe09iZZL_6VhuLHJ4TFd5iGHU6RA7zWS0-lZD0khhDoOeoM1wjhNNFp0WgJBiWtwVZ4KPeDLnwlP4-vTccdTrBK91eY95z6NWyTru--5ButPOqoeE3znobQ-7k5dzoeeQG9Yq4Au34QMd8BuTE9XvctLdY4mlg0vPDpJIPKs2kQpHuzfHhAPC2t3AKScLzw93rm7PxGTQHWoOblncngkZjX/https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAXGI5CDMGP7QX546DNPIVD3WSTFSFANCNFSM6AAAAAAR6GXVXI>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
IMPORTANT NOTICE

This communication is from Deloitte LLP, a limited liability partnership registered in England and Wales with registered number OC303675. Its registered office is 1 New Street Square, London EC4A 3HQ, United Kingdom. Deloitte LLP is the United Kingdom affiliate of Deloitte NSE LLP, a member firm of Deloitte Touche Tohmatsu Limited, a UK private company limited by guarantee (“DTTL”). DTTL and each of its member firms are legally separate and independent entities. DTTL and Deloitte NSE LLP do not provide services to clients. Please see www.deloitte.co.uk/about<https://www.deloitte.co.uk/about> to learn more about our global network of member firms. For details of our professional regulation please see Regulators<https://www2.deloitte.com/uk/en/footerlinks1/regulators-and-provision-service-regulations.html>.

This communication contains information which is confidential and may also be privileged. It is for the exclusive use of the intended recipient(s). If you are not the intended recipient(s), please notify ***@***.******@***.***> and destroy this message immediately. Email communications cannot be guaranteed to be secure or free from error or viruses. All emails sent to or from a @deloitte.co.uk email account are securely archived and stored by an external supplier within the European Union.

You can understand more about how we collect and use (process) your personal information in our Privacy Notice<https://www2.deloitte.com/uk/en/legal/privacy.html>.

Deloitte LLP does not accept any liability for use of or reliance on the contents of this email by any person save by the intended recipient(s) to the extent agreed in a Deloitte LLP engagement contract.

Opinions, conclusions and other information in this email which have not been delivered by way of the business of Deloitte LLP are neither given nor endorsed by it.
